custom-plugins/fictive_studios/admin/printing-area/api/print-area-api.php

```diff
@@ -10,4 +10,41 @@ public function get_print_area_data()
         wp_die();
     }
 
+    public function save_print_area_data()
+    {
+        $height = isset($_POST['height']) ? $_POST['height'] : '';
+        $width = isset($_POST['width']) ? $_POST['width'] : '';
+        global $wpdb;
+        $table_name = $wpdb->prefix . 'print_areas';
+        $wpdb->insert(
+            $table_name,
+            array(
+                'height' => $height,
+                'width' => $width,
+                'user' => get_current_user_id()
+            )
+        );
+        wp_die();
+    }
+
+    public function edit_print_area_data()
+    {
+        $height = isset($_POST['height']) ? $_POST['height'] : '';
+        $width = isset($_POST['width']) ? $_POST['width'] : '';
+        global $wpdb;
+        $table_name = $wpdb->prefix . 'print_areas';
+        $wpdb->update(
+            $table_name,
```

Comment:
add some kind of checks to make sure that no special characters are getting inserted